contrib/test_jenkins.sh

```diff
@@ -878,7 +878,7 @@ run_malloc_hook_gtest() {
 
 	echo "==== Running cuda hooks with far jump, $compiler_name compiler ===="
 	$TIMEOUT env \
-		UCM_BISTRO_FORCE_FAR_JUMP=y \
+		UCX_BISTRO_FORCE_FAR_JUMP=y \
```

Comment:
i think it should be `UCX_MEM_BISTRO_FORCE_FAR_JUMP=y`
```
$ ./build-devel/src/tools/info/ucx_info -c|grep BIST
UCX_MEM_BISTRO_FORCE_FAR_JUMP=n
```

Reply:
Yes, you're right.
Force-pushed the change since it is one line commit.